src/seedu/addressbook/parser/Parser.java

```diff
@@ -97,7 +87,11 @@ public Command parseCommand(String userInput) {
         case ExitCommand.COMMAND_WORD:
             return new ExitCommand();
 
+        case NewestEntryCommand.COMMAND_WORD:
+            return new NewestEntryCommand();
+
         case HelpCommand.COMMAND_WORD: // Fallthrough
+
```

Comment:
Be careful of unnecessary or accidental changes here as it can cause unnecessary merge conflict.